package/gluon-mesh-vpn-core/luasrc/lib/gluon/upgrade/500-mesh-vpn

```diff
@@ -40,10 +43,13 @@ if not uci:get('gluon', 'mesh_vpn') then
 		vpn = 'fastd'
 	elseif unistd.access('/lib/gluon/mesh-vpn/tunneldigger') then
 		vpn = 'tunneldigger'
+	elseif  unistd.access('/lib/gluon/mesh-vpn/wireguard') then
+		vpn = 'wireguard'
```

Comment:
Wireguard does not need to handle this migration case, as it will be introduced after the migration to `gluon.mesh_vpn` and hence doesn't need to be migrated to it.

Reply:
This assumes upgrades from older versions are not supported. I feel this use case is valid however. We have quite old installations out there.